crypto/evp/keymgmt_lib.c

```diff
@@ -101,11 +106,11 @@ void *evp_keymgmt_export_to_provider(EVP_PKEY *pk, EVP_KEYMGMT *keymgmt)
         if (pk->ameth->export_to == NULL)
             return NULL;
 
-        /* Otherwise, simply use it */
-        provkey = pk->ameth->export_to(pk, keymgmt);
+        /* Otherwise, simply use it.  The export_to will tell us what it is */
```

Comment:
This comment seems out of date

Reply:
The first sentence is still valid, but I've removed the second sentence.